.ci/deps.sh

```diff
@@ -67,3 +69,13 @@ julia -e "Pkg.add(\"Lint\")"
 
 # Lua commands
 sudo luarocks install luacheck --deps-mode=none
+
+# Infer commands
+wget -nc -O ~/infer.tar.xz https://github.com/facebook/infer/releases/download/v0.7.0/infer-linux64-v0.7.0.tar.xz
+tar xf ~/infer.tar.xz -C ~/
+cd ~/infer-linux64-v0.7.0
+opam init --y
+opam update
+opam pin add --yes --no-action infer .
+opam install --deps-only --yes infer
+./build-infer.sh java
```

Comment:
Hi, I was checking on how to make this easier.
I found - https://github.com/shaunkawano/circle-ci-android-sample/blob/master/circle.yml#L17
That looks much simpler - can you check it out ?